packages/teams-js/src/public/interfaces.ts

```diff
@@ -1078,9 +1078,15 @@ export enum DevicePermission {
   Media = 'media',
 }
 
+/** @hidden */
+export interface M365ChatLicenseInformation {
+  hasM365ChatLicense: boolean;
```

Comment:
I think I left this comment in the doc, but I think having this return an enum offers more flexibility, rather than a boolean.

Reply:
Oh I thought we will improve upon this design once we have a conformation from the Harmony team. I will follow up with them.

Reply:
IMO we should not have two fields. We should only have the `m365ChatLicenseType` field and app developers should call `copilot.license.getM365ChatLicenseType()` and then they can use that value to take specific actions in their app. 

If we think it's important to have a `copilot.license.hasM365ChatLicense()` function, then we can implement it as something like 
`return runtime.hostVersionsInfo?.m365ChatLicenseInfo?.m365ChatLicenseType !== M365ChatLicenseType.None;`

But I would suspect that the type of license would matter to app developers.